types/tx.go

```diff
@@ -137,3 +137,23 @@ func TxProofFromProto(pb tmproto.TxProof) (TxProof, error) {
 
 	return pbtp, nil
 }
+
+// ComputeProtoOverheadForTxs wraps the transactions in tmproto.Data{} and calculates the size.
+// https://developers.google.com/protocol-buffers/docs/encoding
+func ComputeProtoSizeForTxs(txs []Tx, tx Tx) int64 {
```

Comment:
why have tx Tx as a second parameter when you can add it to `txs`? `ComputeProtoSizeForTxs(txs, tx))`

Reply:
Will open a pr for this change